lms/services/h_api.py

```diff
@@ -93,27 +93,25 @@ def get_user(self, username):
 
     def get_annotations(
         self,
-        audience_userids: List[str],
+        h_userid: str,
         created_after: datetime,
         created_before: datetime,
     ) -> Iterator[dict]:
         """
-        Get an iterator of annotation objects for the specified audience.
+        Get an iterator of annotation objects for the specified h_userid.
 
-        This is an iterator of annotations viewable by _any_ of the provided
-        userids. It is your responsibility to work out who can see what, by
-        knowing which groups users are in.
```

Comment:
This was such a terrible idea for an API design, glad to be able to delete this comment.